employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java

```diff
@@ -37,4 +35,33 @@ public boolean updateEmployee(Employee employee) {
         }
         return result;
     }
+
+    // add employee
+    public EmployeeFacadeImpl(EmployeeDao employeeDao) {
+        this.employeeDao = employeeDao;
+    }
+
+    public boolean addEmployee(Employee employee) {
+        boolean result = false;
+        try {
+            Employee targetEmployee = employeeDao.getEmployeeById(employee.getEmployeeNo());
+            if (targetEmployee != null) {
+                throw new RuntimeException("Employee to add already exists.");
+            }
+            result = employeeDao.addEmployee(employee);
+        } catch (Exception e) {
+            throw new RuntimeException(e.getMessage());
+        }
+
+        if (result) {
+            System.out.println("Employee added successfully.");
```

Comment:
move these messages in Facade. leave messages for catching exceptions

Reply:
Sir di ko po nagets masyado, ano po ibig sabihin nito sir? Thank you po.